common/src/main/java/me/pieking1215/invmove/InvMove.java

```diff
@@ -244,54 +262,18 @@ public void onInputUpdate(/*$ Input {*/ClientInput/*$}*/ input
             wasSneaking = input.keyPresses.shift();
             //?} else
             /*wasSneaking = input.shiftKeyDown;*/
+        } else {
+            // tick keybinds (since opening the ui unpresses all keys)
+            tickKeybinds();
```

Comment:
Moving this here makes it update all keybinds even if the screen disables movement
I see why you had to do it though, since you need the raw sneak key input to do the new mode even when movement is disabled
But I think we could just have it only update the sneak key in that case, I'll do a followup commit